src/core/evaluator.js

```diff
@@ -2656,20 +2654,26 @@ class PartialEvaluator {
       textContentItem.str.length = 0;
     }
 
-    function enqueueChunk() {
+    function enqueueChunk(batch = false) {
       const length = textContent.items.length;
-      if (length > 0) {
-        sink.enqueue(textContent, length);
-        textContent.items = [];
-        textContent.styles = Object.create(null);
+      if (length === 0) {
+        return;
+      }
+      if (batch && length < 5) {
```

Comment:
We could likely bump this up to e.g. `10` instead (that'd bring the `postMessage`-reduction to ~75%) without too much problem, but I figured that starting small might not be a bad idea.